parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -5551,6 +5553,58 @@ SqlGetDiagnosticsParam SqlGetDiagnosticsParam() :
     { return new SqlGetDiagnosticsParam(s.end(this), name, value); }
 }
 
+SqlCall FirstLastValue() :
+{
+    final boolean first;
+    final SqlNode value;
+    final SqlNode over;
+    NullOption nullOption = NullOption.UNSPECIFIED;
+    final SqlCall firstLastCall;
+}
+{
+    (
+        <FIRST_VALUE> {
+            first = true;
```

Comment:
nit: Instead of using a boolean here, maybe we could define a final SqlAggFunc variable that hold either SqlStdOperatorTable.FIRST_VALUE or SqlStdOperatorTable.LAST_VALUE. So we can reduce the createCall() logic into one single line.